src/components/DocRoot/index.js

```diff
@@ -130,14 +130,14 @@ export default function DocRoot() {
         <li><span className="doc-section">Reference</span>. Detailed list of out-of-the-box
           instrumentation provided by Micrometer.
           <ul>
-            <li><span className="doc-section">Spring Boot</span>. As of Spring Boot 2.0.0.M5, Micrometer is the
+            <li><span className="doc-section">Spring Boot</span>. As of Spring Boot 2.0, Micrometer is the
```

Comment:
I think we're far enough away as to not need to be this specific anymore - users shouldn't use old milestones and we don't want to inadvertently encourage that.